apps/src/weblab/dialogs/UploadErrorDialog.story.js

```diff
@@ -2,19 +2,13 @@ import React from 'react';
 import {action} from '@storybook/addon-actions';
 import UploadErrorDialog from './UploadErrorDialog';
 
-export default storybook => {
-  storybook
-    .storiesOf('Weblab/dialogs/UploadErrorDialog', module)
-    .addStoryTable([
-      {
-        name: 'default',
-        story: () => (
-          <UploadErrorDialog
-            isOpen
-            handleClose={action('close')}
-            hideBackdrop
-          />
-        )
-      }
-    ]);
+export default {
+  title: 'UploadErrorDialog',
```

Comment:
I think we want to keep the directory structure here - `Weblab/dialogs/UploadErrorDialog`. That adds 'chapters' to the storybook that helps group features. Example screenshot of how it looks for MakerToolkit.
![Screenshot from 2022-09-12 09-43-54](https://user-images.githubusercontent.com/2959170/189710393-f26cfb70-55a4-411d-bc1d-8f29fd5b009b.png)

Reply:
Nevermind! Ignore this! Maddie just let me know we are dropping this organization practice! Sorry for the false alarm!

Reply:
Thanks for checking on this!

Reply:
+1, thank you, erin! i would like us to use this organization structure in the future, but the way we have it currently is haphazard and doesn't really make any sense, so i think we do away with it for now since it's easy to do in the future